exercises/joseph_torsney/ex02_oo_basics/src/rect.h

```diff
@@ -0,0 +1,25 @@
+#ifndef RECT_H
+#define RECT_H
+
+#include "shape.h"
+
+class Rect: public Shape {
+  double side1;
+  double side2;
+
+  public:
+    Rect(double a, double b): Shape("rect", 4) {
+      side1 = a;
+      side2 = b;
+    }
+
+    double getArea() {
```

Comment:
As these methods are overriding a virtual function you can add the [`override`](https://en.cppreference.com/w/cpp/language/override) keyword to the function definition which will check at compile time if any parameters are incorrect and doesn't override as intended.